runtime/startup_aux.c

```diff
@@ -118,6 +130,31 @@ void caml_parse_ocamlrunparam(void)
         break;
       case 'V': scanmult (opt, &params.verify_heap); break;
       case 'W': scanmult (opt, &caml_runtime_warnings); break;
+      case 'X': {
+        char *name = opt;
+        while (*opt != '\0') {
+          if (*opt == '=') {
+            if (opt - name == strlen("help") &&
+                memcmp(name, "help", opt - name) == 0) {
+              fprintf(stderr, "Known GC tweaks:\n");
+              caml_print_gc_tweaks();
+            } else {
+              uintnat* p = caml_lookup_gc_tweak(name, opt - name);
+              if (p == NULL) {
+                fprintf(stderr, "Ignored unknown GC tweak '%.*s'. "
+                        "Use 'Xhelp=1' to list known tweaks\n",
```

Comment:
```suggestion
                        "Use 'Xhelp' to list known tweaks\n",
```
(combined with the suggestion above, as `Xhelp=0` also displays the list)